db/range_del_aggregator_bench.cc

```diff
@@ -203,10 +213,27 @@ int main(int argc, char** argv) {
 
       auto range_del_iter =
           rocksdb::MakeRangeDelIterator(persistent_range_tombstones);
-      rocksdb::StopWatchNano stop_watch_add_tombstones(rocksdb::Env::Default(),
-                                                       true /* auto_start */);
-      range_del_agg.AddTombstones(std::move(range_del_iter));
-      stats.time_add_tombstones += stop_watch_add_tombstones.ElapsedNanos();
+      fragmented_range_tombstone_lists.emplace_back(
+          new rocksdb::FragmentedRangeTombstoneList(
+              rocksdb::MakeRangeDelIterator(persistent_range_tombstones), icmp,
```

Comment:
I was thinking of only having a mode where tombstones are created in order, since having a sorted list should be the common case (i.e., the sorting should only happen once). But yeah, this PR is already large enough, so I'll do this in a follow-up PR. Thanks for noticing this!